src/main/java/com/github/fabienrenaud/jjb/stream/UsersStreamSerializer.java

```diff
@@ -1149,4 +1156,90 @@ public String underscore_java(final Users obj) throws IOException {
         }
         return jso;
     }
+
+    @Override
+    public Value purejson(Users obj) throws IOException {
+        final Map<Parser.Value, Parser.Value> map = new LinkedHashMap<>();
+        final List<Parser.Value> arr = new ArrayList<>();
```

Comment:
* create the array after the if check,
 * init the capacity of the array with obj.users.size()